python/SetupPython.cmake

```diff
@@ -35,9 +35,11 @@ else()
   endif()
 endif()
 
-execute_process(COMMAND ${CMAKE_COMMAND} -E env --unset=PIP_REQUIRE_VIRTUALENV ${Python_EXECUTABLE}
-  -m pip install --target=${ENERGYPLUS_DIR}/python_standard_lib --upgrade -r ${PROJECT_SOURCE_DIR}/python/requirements.txt
-)
+if(BUILD_CLI)
+  execute_process(COMMAND ${CMAKE_COMMAND} -E env --unset=PIP_REQUIRE_VIRTUALENV ${Python_EXECUTABLE}
+    -m pip install --target=${ENERGYPLUS_DIR}/python_lib --upgrade -r ${PROJECT_SOURCE_DIR}/python/requirements.txt
+  )
+endif()
```

Comment:
I'm wrapping this into a BUILD_CLI, as a way to not be forced to drop our PyPi bindings for Python 3.8 yet.

But that means you're very likely going to be unable to build a regular installers with BUILD_CLI and BUILD_PYTHON_BINDINGS enabled. I haven't tested TBH, so maybe it'll still work.